### PR TITLE
Add LJSpeech corpus

### PR DESCRIPTION
**Pull request by JackTemaki**
Add a download and a bliss creation job for the LJSpeech corpus.

### CODE REVIEW

File: util.py [hunk 1]
```diff
@@ -252,3 +252,16 @@ def create_executable(filename, command):
         | stat.S_IXGRP
         | stat.S_IXOTH,
     )
+
+
+def check_file_checksum(filename, reference_checksum):
```
Comment (curufinwe) [inline util.py:257]: if sha256 is hardcoded in the function it should be added to the name (for better readability). i.e. `check_file_sha256sum`

**Review by curufinwe — CHANGES_REQUESTED** [review]

File: datasets/ljspeech.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+__all__ = [
+    "DownloadLJSpeechCorpusJob",
+]
```
Comment (Atticus1806) [inline datasets/ljspeech.py:3]: Does it make sense to just hase the Download Job here? Maybe also add the other job.

File: datasets/ljspeech.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        self.out_ljspeech_folder = self.output_path("LJSpeech-1.1")
+        self.out_audio_folder = self.output_path("LJSpeech-1.1/wavs")
+        self.out_metadata = self.output_path("LJSpeech-1.1/metadata.csv")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        subprocess.check_call(
+            ["wget", "https://data.keithito.com/data/speech/LJSpeech-1.1.tar.bz2"]
```
Comment (michelwi) [inline datasets/ljspeech.py:29]: Do we have the option to download to a node-local temp location instead of the work folder?

Comment (curufinwe) [reply datasets/ljspeech.py:29]: In the code right now not, but in principle I see no problem implementing it that way unless we want to be able to resume downloads and add the corresponding flags to wget.

**Pull request merged**